packages/api/package.json

```diff
@@ -9,26 +9,22 @@
   "dependencies": {
     "@esm2cjs/p-queue": "^7.3.0",
     "@hyperdx/common-utils": "^0.5.0",
-    "@hyperdx/lucene": "^3.1.1",
     "@hyperdx/node-opentelemetry": "^0.8.2",
     "@hyperdx/passport-local-mongoose": "^9.0.1",
     "@opentelemetry/api": "^1.8.0",
     "@opentelemetry/host-metrics": "^0.35.5",
     "@opentelemetry/sdk-metrics": "^1.30.1",
     "@slack/webhook": "^6.1.0",
     "@types/node": "^22.15.18",
-    "axios": "^1.6.2",
     "compression": "^1.7.4",
     "connect-mongo": "^4.6.0",
     "cors": "^2.8.5",
     "cron": "^3.5.0",
     "date-fns": "^2.28.0",
-    "date-fns-tz": "^2.0.0",
```

Comment:
I setup [knip](https://knip.dev/) at my last job - it was super helpful for this.

I can make a ticket if you think its helpful